client/src/pages/private/SiteParticipantsTable.js

```diff
@@ -275,7 +275,7 @@ export default ({ id, siteId, onArchiveParticipantAction, stale, setStale }) =>
       },
       body: JSON.stringify({
         participantId,
-        site: parseInt(id),
+        site: parseInt(siteId),
```

Comment:
I found out from Pushan that `+siteId` also works instead of passing it through parseInt.. just a lil bit of sytactic sugar

Reply:
Neat. Every day I learn something new. I'll implement it.